cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -421,3 +431,19 @@ producerAddresses nt =
    remoteOrNode ra = case remoteAddressToNodeAddress ra of
                        Just na -> Right na
                        Nothing -> Left ra
+
+-- |
+-- Possibly return a SOCKET_UNIX socket for NodeToClient communication
+-- and a list of SOCKET_STREAM sockets for NodeToNode communication.
+-- The SOCKEt_UNIX socket should be defined first in the `.socket` file.
```

Comment:
I would rephrarse it as: in the `.socket` systemd file - it might be not clear when reading haddocks.